src/electronic-ids/pcsc/FinEID.cpp

```diff
@@ -66,10 +66,9 @@ namespace electronic_id
 
 byte_vector FinEIDv3::getCertificateImpl(const CertificateType type) const
 {
+    transmitApduWithExpectedResponse(*card, SELECT_MAIN_AID);
     return electronic_id::getCertificate(
-        *card,
-        {SELECT_MAIN_AID,
-         type.isAuthentication() ? SELECT_AUTH_CERT_FILE : SELECT_SIGN_CERT_FILE_V3});
+        *card, type.isAuthentication() ? SELECT_AUTH_CERT_FILE : SELECT_SIGN_CERT_FILE_V3);
```

Comment:
The code was more DRY before. Is there a reason for moving `transmitApduWithExpectedResponse(*card, SELECT_MAIN_AID)` out of the commands passed to `getCertificateImpl()`?

Reply:
The clang formatting and being only case where is two select apdu-s id argue, this is DRY